p9-presentation/sections/_methodology.qmd

```diff
@@ -1,7 +1,494 @@
+<br>
+The process itself follows the steps seen here.<br>
+<br>
+First we calculate the leverage and spectral residuals from the scores and loadings of a PLS model.<br>
+<br>
+<!-- Leverage is a measure of how far the independent variable values of an observation are from other observations. High-leverage points are outliers w.r.t the independent variables, as they have no neighboring points in space.<br>
+<br>
+Spectral residuals measure how well a given spectrum is explained by the model. -->
+Then we calculate the Mahalanobis distance for each point in a matrix of the combined leverage and spectral residuals.<br>
+<br>
+We obtain a critical value that is based on a chi-square distribution for a 97.5% confidence interval given 2 degrees of freedom.<br>
+<br>
+We compare the distances against this critical value, using it as a threshold, where distances above this threshold are classified as outliers.<br>
+<br>
+Then we remove outliers from the dataset, retrain a PLS model, and then repeat the process.
+:::
+
+## Outlier Removal {.unlisted}
+::: {.r-stack}
+![](/static/methodology/FeOT_Full_1.png)
+
+![](/static/methodology/FeOT_Full_2.png){.fragment fragment-index=1}
+
+![](/static/methodology/FeOT_Full_3.png){.fragment fragment-index=2}
+
+![](/static/methodology/FeOT_Full_4.png){.fragment fragment-index=3}
+
+![](/static/methodology/FeOT_Full_5.png){.fragment fragment-index=4}
+:::
+
+::: {.notes}
+The process itself is easily seen as we step through these influence plots.<br>
+<br>
+As you can see, those items with both a high error and leverage are classified as outliers and are promptly removed.
+:::
+
+## MOC Pipeline {auto-animate="true"}
+::: {style="width=100%;"}
+![](/static/methodology/pipeline.png){fig-align="center" width="35%"}
+:::
+
+::: {data-id="moc-box"}
+<div style="position: absolute; top: 34.45%; left: 51.25%; width: 18%; height: 35.25%; border: 2px solid red; box-sizing: border-box; pointer-events: none; border-radius: 10px;"></div>
+:::
+
+## ICA Preprocessing
+:::: {.columns}
+::: {.column width="50%"}
+- Similar preprocessing to PLS-SM, except:
+    - No averaging
+    - No compositional range filtering
+    - Uses only one location per sample
+:::
+::: {.column width="50%"}
+![ICA Preprocessing](static/methodology/ica_preprocessing.png){width="55%"}
+:::
+::::
+
+::: {.notes}
+I won't go into much detail about the preprocessing stage in the ICA phase.<br>
+The process mostly follows the same steps as in PLS-SM, except we don't filter for any compositional ranges here.<br>
+<br>
+We also do not average the shots, and instead preserve the intensity values for each shot, except the first five.<br>
+<br>
+Finally, we don't use all five location datasets for each sample. That is left as an experiment, which Ivik will go into details about. We chose to do it this way because it was unclear whether the original authors used all location datasets or just one per sample.
```

Comment:
```suggestion
Finally, we don't use all five location datasets for each sample. That is left as an experiment, which Ivik will go into detail about. We chose to do it this way because it was unclear how the original authors used all location datasets.
```
IMO it was clear they used all five - but it wasn't clear how they did it. That's also what we said in our report.